src/Components/APICalls.js

```diff
@@ -2,34 +2,27 @@ import { useQuery } from "@tanstack/react-query";
 
 const fiveMinutesMs = 1000 * 60 * 5;
 
+const apiUrl = `https://api-jet-lfoguxrv7q-uw.a.run.app`;
+
 export async function APISearch(inputValue) {
-  try {
-    let response = await fetch(
-      `https://api-jet-lfoguxrv7q-uw.a.run.app/anime/search?query=${inputValue}`,
-      { mode: "cors" }
-    );
-    let responseJson = await response.json();
-    let temp = [];
-    responseJson.items.map((item, index) => temp.push(item));
-    return temp;
-  } catch (error) {
-    console.log(error);
-  }
+  let response = await fetch(`${apiUrl}/anime/search?query=${inputValue}`, {
+    mode: "cors",
+  });
+  let responseJson = await response.json();
```

Comment:
Should we call `await handleErrors(response);` here, like in other API functions, to make sure that non-OK responses end up throwing?

Reply:
Great call! Totally forgot we had broken out that error handling function.